tests/llm_response_parsing_test.py

```
@@ -0,0 +1,364 @@
+"""Tests for robust LLM response parsing."""
+
+import pytest
```

Ruff (0.12.2)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)